examples/sitepurpose.yml

```diff
@@ -0,0 +1,36 @@
+-
+  _valid: true
+  _name: 'grid-founding-member'
+  _data:
+    type: 'purchase'
+    label: 'Become a Founding Member'
+    cta: 'f6a4bd8d-55ec-4c3a-9ccd-9c5f3df80802'
```

Comment:
Purposes don't have IDs. The `cta` key is used when associating a site purpose with a purchase CTA on Reserve. Like for example Grid's founding membership purchase